encrypt_util.go

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) 2021 Snowflake Computing Inc. All right reserved.
+
+package gosnowflake
+
+//lint:file-ignore U1000 Ignore all unused code
+
+import (
+	"bytes"
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"encoding/base64"
+	"encoding/json"
+	"strconv"
+)
+
+const (
+	blockSize = int(aes.BlockSize / 8)
```

Comment:
Can you add comments what are we doing here?

Reply:
Turns out the the AES package's BlockSize is actually in bytes, which is what I wanted to achieve.

Reply:
Yup that's what I thought and hence asked.